examples/controlnet/train_controlnet.py

```diff
@@ -541,6 +538,12 @@ def parse_args(input_args=None):
             " more information see https://huggingface.co/docs/accelerate/v0.17.0/en/package_reference/accelerator#accelerate.Accelerator"
         ),
     )
+    parser.add_argument(
+        "--controlnet_conditioning_embedding_out_channels",
+        type=int,
+        nargs="+",
+        default=None,
+    )
```

Comment:
Needed for running the script against the test dummy stable diffusion pipeline's encoder

Reply:
Hmm can't we better use a correct dummy controlnet & stable diffusion model config use here? Not keen on adding this just because of testing. Using this controlnet: https://huggingface.co/hf-internal-testing/tiny-controlnet with https://huggingface.co/hf-internal-testing/tiny-controlnet/blob/b9292c9b1f54a486cfd440be2c0300b189754331/README.md?code=true#L3

Reply:
I agree, I'd rather not have a new arg (which can be confusing to users) just for the sake of testing.

Reply:
hm, ok will see if can avoid!